components/link-list/LinkList.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <dl class="link-list">
+    <template v-for="link in links">
+        <dt v-if="link.slug" v-bind:class="`h3 invert ${link.slug}`">
+          <nuxt-link v-bind:to="link.path">{{ link.title }}</nuxt-link>
+        </dt>
```

Comment:
link list is not a definition list `dl`. Could be an unordered list `ul` inside a `nav`, but since there are not that many links there, adding a single `nav` and an `nuxt-link` should be enough